### PR TITLE
Change from hardcoded strings to String Resources (Resolves #269)

### LINKED ISSUE #269: Internationalization
User interface text should be internationalized using strings with platform language conditions. Event description translations are not as feasible.
- [ ] Internationalize UI strings
- [ ] Internationalize dynamic content

### PR DESCRIPTION
**Pull request by Nikoshi**
I've extracted the hardcoded strings into String Resources,
please comment if i did miss something

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,16 +1,273 @@
...
+    <string name="dealers">Dealers</string>
+    <string name="dealer_name">Dealer Name</string>
+    <string name="dealer_subname">Subname</string>
+    <string name="dealer_about_artist">About the Artist</string>
+    <string name="dealer_about_art">About the Art</string>
+    <string name="dealer_telegram_handle">%1$s @ Telegram</string>
+    <string name="dealer_twitter_handle">%1$s @ Twitter</string>
+    <string name="dealer_location_and_availability">Location &amp; Availability</string>
+    <string name="dealer_only_present_on">{fa-exclamation-triangle 24sp} Only present on: %1$s</string>
+    <string name="dealer_located_in_the_after_dark">{fa-moon-o 24sp}  Located in the After Dark Dealers Den</string>
```
Comment (Pinselohrkater) [inline app/src/main/res/values/strings.xml]: (Nitpick) IMHO the FA icons shouldn't be part of the translation.

Comment (Nikoshi) [reply app/src/main/res/values/strings.xml]: Okay make sense, I'll fix these later when I'm at home

File: app/src/main/kotlin/org/eurofurence/connavigator/ui/ActivityRoot.kt [hunk 2]
```diff
@@ -522,17 +523,17 @@ class ActivityRoot : AppCompatActivity(), RootAPI, SharedPreferences.OnSharedPre
         navSubtitle.text = RemotePreferences.eventSubTitle
         // On con vs. before con. This should be updated on day changes
         if (days <= 0)
-            navDays.text = "Day ${1 - days}"
+            navDays.text = getString(R.string.misc_current_day, 1 - days)
         else
-            navDays.text = "Only $days days left!"
+            navDays.text = getString(R.string.misc_days_left, days)
```
Comment (Requinard) [inline app/src/main/kotlin/org/eurofurence/connavigator/ui/ActivityRoot.kt]: same with the formatted strings

File: app/src/main/kotlin/org/eurofurence/connavigator/util/Formatter.kt [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+import android.content.Context
 import io.swagger.client.model.DealerRecord
 import io.swagger.client.model.EventRecord
+import org.eurofurence.connavigator.R
 import java.util.*
 
 /**
  * Created by David on 11-5-2016.
  */
 object Formatter {
```
Comment (Requinard) [inline app/src/main/kotlin/org/eurofurence/connavigator/util/Formatter.kt]: The formatter class is really old and shouldn't be used anymore, Maybe a new issue can be created where this class is instead replaced by extension functions?

**Pull request merged**